packages/sqlite-storage/example/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-native-example-sqlitestorage",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "build:android": "npm run mkdist && react-native bundle --entry-file ../../../../../../Desktop/sqlite-storage/example/index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist && react-native build-android --extra-params \"--no-daemon --console=plain -PreactNativeArchitectures=arm64-v8a\"",
+    "build:ios": "npm run mkdist && react-native bundle --entry-file ../../../../../../Desktop/sqlite-storage/example/index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist && react-native build-ios --scheme AwesomeLibraryExample --mode Debug --extra-params \"-sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO\"",
```

Comment:
Are the entry file paths correct? They look like they reference something locally on your machine?

Reply:
Yes, lots of directory moving 😄